src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,22 @@
+.stars {
+  display: flex;
+  align-items: center;
+
+  @for $i from 1 through 5 {
+    &--#{$i} &__star:nth-child(-n + #{$i}) {
+      background-image: url('../images/star-active.svg');
+    }
+  }
```

Comment:
Consider using a more descriptive variable name instead of `$i` in the `@for` loop for better readability. For instance, `$star-index` or `$active-stars` would more clearly convey the purpose of the variable.